L1Trigger/TrackerTFP/plugins/ProducerAS.cc

```diff
@@ -0,0 +1,100 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+
+#include <string>
+
+using namespace std;
+using namespace edm;
+using namespace trackerDTC;
+
+namespace trackerTFP {
+
+  /*! \class  trackerTFP::ProducerAS
```

Comment:
Since ProducerAS is only required to do analysis, its a shame that it's cluttering up the plugins/ directory, instead of being in test/ like all your other analysis code.

Reply:
done